server/utils/messageUtil.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+module.exports = {
+    succuss: 'success',
```

Comment:
naming typo succus -> success, not a big deal but if we can prevent ambiguity,  it will look good!